deploy/bin/update-worker-types.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+const bluebird = require('bluebird');
+const workerTypes = require('../etc/worker-types.json');
+const amis = require('../../docker-worker-amis.json');
+const jsonfile = bluebird.Promise.promisifyAll(require('jsonfile'));
+const utils = require('./worker_type_utils');
+
+/*
+ * Write worker-types configuration to a json file
+ */
+async function backupWorkerTypes(client) {
+  console.log('Creating worker-types backup file.');
+  const names = await client.listWorkerTypes();
+  const workerTypes = names.map(name => {
+    const w = new utils.WorkerType(client, name);
+    return w.workerType();
+  });
+
+  await Promise.all(workerTypes).then(
```

Comment:
The Promise.all() might read better when initalizing workerTypes than here.  Took me a second to realize that w.workerType() is actually a promise.

Reply:
Hrm, I think that expression is already too complex, tbh.

Reply:
fair enough. :)